store/sql_channel_store.go

```diff
@@ -37,6 +38,7 @@ func NewSqlChannelStore(sqlStore *SqlStore) ChannelStore {
 
 func (s SqlChannelStore) UpgradeSchemaIfNeeded() {
 	s.CreateColumnIfNotExists("Channels", "ExtraUpdateAt", "TotalMsgCount", "bigint(20)", "0")
+	s.CreateColumnIfNotExists("Channels", "CreatorId", "ExtraUpdateAt", "varchar(26)", "")
```

Comment:
Yup, when that process is figured out we will have this and any new columns added to the 0.7.0 SQL script (or whatever we use)